### PR TITLE
add min_swap_amount logic

### PR DESCRIPTION
**Pull request by toprince**
as title

### CODE REVIEW

File: programs/woofi/src/instructions/swap.rs [hunk 1]
```diff
@@ -96,6 +96,12 @@ pub struct Swap<'info> {
 }
 
 pub fn handler(ctx: Context<Swap>, from_amount: u128, min_to_amount: u128) -> Result<()> {
+    // skip swap when from amount less than min swap amount
+    // do not fail directly, due to consider third party call will fail in this situration
+    if from_amount < ctx.accounts.woopool_from.min_swap_amount {
```
Comment (fb-alexcq) [inline programs/woofi/src/instructions/swap.rs:101]: < or <=
 
 what happened if from_amount == 0?

File: programs/woofi/src/instructions/swap.rs [hunk 2]
```diff
@@ -96,6 +96,12 @@ pub struct Swap<'info> {
 }
 
 pub fn handler(ctx: Context<Swap>, from_amount: u128, min_to_amount: u128) -> Result<()> {
```
Comment (fb-alexcq) [inline programs/woofi/src/instructions/swap.rs:98]: Do we also check this min swap amount in `query` function?

File: programs/woofi/src/instructions/swap.rs (hunk 1 above)
Comment (toprince) [reply programs/woofi/src/instructions/swap.rs:101]: ok, use <= can skip setting this min_swap_amount initially
will use <=

File: programs/woofi/src/instructions/swap.rs (hunk 2 above)
Comment (toprince) [reply programs/woofi/src/instructions/swap.rs:98]: will add

**Pull request merged**